practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>"General architecture of a recommender systems" -&gt; system</p><p> </p><p>Is the GPU strictly necessary for the prac? Might make it tricky for people without GPUs on their laptops</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Not it's not actually especially for the smaller datasets. I've updated it now.</p>